### PR TITLE
Newq branch

### PR DESCRIPTION
**Pull request by ayushk12**
added next question

### CODE REVIEW

File: javascript/answers/ayushk12-q22.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            <h4>
+                <h4>Explain window.onload and onDocumentReady?</h4>
+            </h4>
+        </div>
+        <br>
+        <br>
+        <div>
+            <strong>Answer:</strong>
+            <p>
+                The onload function is not run until all the information on the page is loaded. This leads to a substantial delay before any code is executed.
```
Comment (shivamyth) [inline javascript/answers/ayushk12-q22.html]: Need an example of the same.

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/ayushk12-q22.html (hunk 1 above)
Comment (ayushk12) [reply javascript/answers/ayushk12-q22.html]: added example

File: javascript/answers/ayushk12-q22.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        gtag('config', 'UA-157837769-1');
+    </script>
+
+    <title>
+        JavaScript | InterQues | Coding Ninjas
+    </title>
+
+    <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png" />
+
+    <meta name="description" content="What is the value of !'bang'?">
```
Comment (shivamyth) [inline javascript/answers/ayushk12-q22.html]: content in meta tag should be question title.

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/ayushk12-q22.html (hunk 2 above)
Comment (ayushk12) [reply javascript/answers/ayushk12-q22.html]: changes done

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@ayushk12 Hey, please take a fresh pull (from upstream development) and then commit your changes.
Also, you have made changes in the `readme.md` file, so please remove them.

Please make the requested changes ASAP as this PR will be closed in 24 hours if no changes are made.

File: README.md [hunk 3]
```diff
@@ -64,3 +64,4 @@ For making any changes (from your local system), just following the following st
 Respect fellow developers, everyone might be at a different stage of skill levels 
  Encourage fellow developers, pass on an opportunity to them, if they've asked something simple, just point them to the right direction in as polite a manner as possible 
  Thank fellow developers if they help you out in any way 
+
```
Comment (dakshkhetan) [inline README.md]: Remove this change

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
You still haven’t removed the unnecessary changes such as README.md file change (which can be seen under files changed tab above). Also, there's a subfolder named InterQues is created, so remove that too.

File: README.md (hunk 3 above)
Comment (ayushk12) [reply README.md]: modified all the changes please review again

**Comment by dakshkhetan** [comment]
@ayushk12 As you can see, there are still so many unnecessary changes which are not removed.
I suggest you to start the whole process again, i.e. forking, then cloning, creating a new branch, and then pushing the code and finally raise a new PR.

**Pull request closed without merging**